### PR TITLE
Upgrade GitSync and fix environment variables to support username & p…

### PR DESCRIPTION
**Pull request by cedbossneo**
…assword combination

### CODE REVIEW

File: pkg/apis/airflow/v1alpha1/airflow.go [hunk 1]
```diff
@@ -1203,7 +1203,7 @@ func (s *GitSpec) container(volName string) (bool, corev1.Container) {
 		VolumeMounts: []corev1.VolumeMount{
 			{
 				Name:      volName,
-				MountPath: "/git",
+				MountPath: "/tmp/git",
```
Comment (barney-s) [inline pkg/apis/airflow/v1alpha1/airflow.go]: why this change ? Is it needed to work ?

**Comment by cedbossneo** [comment]
The new release of git-sync set the home directory to /tmp/git  instead of
/git. Maybe thereis a way To set chat to /git with the workingdir parameter
in the pod spec but i didn't tried
